x/projects/keeper/project.go

```diff
@@ -326,6 +327,20 @@ func (k Keeper) SetProjectPolicy(ctx sdk.Context, projectIDs []string, policy *p
 				utils.Attribute{Key: "block", Value: ctxBlock},
 			)
 		}
+
+		projectIDsStr += projectID + ", "
+	}
+
+	details := map[string]string{
+		"creator":     key,
+		"project_ids": projectIDsStr,
+		"policy":      policy.String(),
+	}
+
+	if setPolicyEnum == types.SET_ADMIN_POLICY {
+		utils.LogLavaEvent(ctx, k.Logger(ctx), types.SetAdminPolicyEventName, details, "set admin policy successfully")
+	} else if setPolicyEnum == types.SET_SUBSCRIPTION_POLICY {
+		utils.LogLavaEvent(ctx, k.Logger(ctx), types.SetSubscriptionPolicyEventName, details, "set admin policy successfully")
```

Comment:
Is it supposed to be:
```suggestion
		utils.LogLavaEvent(ctx, k.Logger(ctx), types.SetSubscriptionPolicyEventName, details, "set subscription policy successfully")
```

Reply:
yes. nice catch. see here: https://github.com/lavanet/lava/commit/189b8756e705e6ccd58fd7e92b7231078bd5e5e0